homeTask2/src/hooks/useUsers.tsx

```diff
@@ -0,0 +1,40 @@
+import { useState } from "react";
+import { UserItem } from "../components/types";
+
+const useUsers = () => {
+    const [userHistoryList, setUserHistoryList] = useState(JSON.parse(localStorage.getItem("UserHistory") || "[]"));
+    const addUser = (event: any, userInfo: UserItem) => {
+        event.preventDefault();
```

Comment:
event.preventDefault should be handled in the component, it is not needed in hook